platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/isolated/IsolatedProjectsCompositeBuildParallelConfigurationIntegrationTest.groovy

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.cc.impl.isolated
+
+import org.gradle.test.fixtures.file.TestFile
+import org.gradle.test.fixtures.server.http.BlockingHttpServer
+import org.junit.Rule
+
+class IsolatedProjectsCompositeBuildParallelConfigurationIntegrationTest extends AbstractIsolatedProjectsCompositeBuildIntegrationTest {
+
+    @Rule
+    BlockingHttpServer server = new BlockingHttpServer(5_000)
+
+    def setup() {
+        server.start()
+    }
+
+    def "plugin builds are being configured in defined order despite plugins from them were requested concurrently"() {
```

Comment:
❓ Why independent plugin builds are not configured in parallel? Is it because of `BuildLogicBuildQueue`?

Reply:
Because the plugin resolution from included builds is a plain `for loop`:
See:
1. https://github.com/gradle/gradle/blob/98ba4606a2fa779ed49ed1c7250ed7170018f100/subprojects/composite-builds/src/main/java/org/gradle/composite/internal/plugins/CompositeBuildPluginResolverContributor.java#L137

2. https://github.com/gradle/gradle/blob/98ba4606a2fa779ed49ed1c7250ed7170018f100/subprojects/composite-builds/src/main/java/org/gradle/composite/internal/plugins/CompositeBuildPluginResolverContributor.java#L127

Reply:
I see, thanks